RunJavaUtils.py

```diff
@@ -49,7 +49,7 @@ def Compare_Lines(goldenList, studentList):
         if (i >= len(goldenList) or i >= len(studentList)):
             mismatch.append("->")
             anyMismatch=True
-        elif (goldenList[i] != studentList[i]):
+        elif (goldenList[i].strip() != studentList[i].strip()):
```

Comment:
I think this will also cause *leading* whitespace errors to incorrectly match, right?

Reply:
I intended this one.  Sometimes the kids put an extra leading space in front of everything, which should still be valid as long as everything aligned.